packages/redux-devtools-extension/package.json

```diff
@@ -42,6 +42,7 @@
     "@typescript-eslint/parser": "^5.12.1",
     "eslint": "^8.10.0",
     "eslint-config-prettier": "^8.4.0",
+    "immutable": "^4.0.0",
```

Comment:
I think this needs to go in `dependencies` since the declaration file will depend on it. Although I'm not super happy about adding `immutable` to the dependencies of this package. This package is meant to be extremely light and adding Immutable.js to the dependencies leaves a bad taste in my mouth.

Can you also build it and double-check that immutable is not imported in the JS build output?

Reply:
Ok, moved it to `dependencies`.

Double-checked: it's not in the JS build output, the build's still the same size.